server/server.js

```diff
@@ -0,0 +1,20 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+const Photos = require('../database/photomodel');
+
+const app = express();
+app.use(bodyParser.json());
+
+const port = 3000;
+
+app.get('/api/restaurants/:id/photos', (req, res) => {
+  console.log(req.params.id);
```

Comment:
I'd remove unnecessary console logs as well. If the message was more descriptive, then I'd keep it.